templates/base.lds

```diff
@@ -104,7 +104,7 @@ SECTIONS
 {% else %}
```

Comment:
This `{% else %}` clause was technical debt I added, and forgot to remove after you had  `weak`-ened `__meta_eccscrub_bit` . Thanks!